storage/series.go

```diff
@@ -227,25 +230,47 @@ func (s *seriesToChunkEncoder) Iterator() chunks.Iterator {
 	mint := int64(math.MaxInt64)
 	maxt := int64(math.MinInt64)
 
+	i := 0
 	seriesIter := s.Series.Iterator()
 	for seriesIter.Next() {
+		// Create a new chunk if too many samples in the current one
```

Comment:
Also in other places where missed.
```suggestion
		// Create a new chunk if too many samples in the current one.
```